networkx/generators/classic.py

```diff
@@ -681,16 +681,16 @@ def trivial_graph(create_using=None):
     return G
 
 
-def turan_graph(n,r):
+def turan_graph(n, r):
     """ Return the Turan Graph
     
     The Turan Graph is a complete multipartite graph on `n` vertices
     with `r` partitions with the property that it has the maximum number
     of edges for any graph with the same number of vertices and partitions.
 
-    Given n and r, we get a complete multipartite graph with `r-(n mod r)`
-    partitions of size `n/r`, rounded down, and `n mod r` partitions of size
-    `n/r+1`, rounded down.
+    Given `n` and `r`, we generate a complete multipartite graph with
+    :math:`r-(n \mod r)` partitions of size :math:`n/r`, rounded down, and
```

Comment:
Let's correct the terminology while we're making a change to the documentation here: a "partition" is a set of sets, each of which may be called a "block", a "part", or a "cell". So this should be changed to say "r - n mod r blocks" instead of "r - n mod r partitions". (I choose "block" to match the docstring for `complete_mulitpartite_graph`.) This needs to be changed in the paragraph above as well.